Tasks/GitHubReleaseV0/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
         "Major": 0,
         "Minor": 0,
-        "Patch": 4
+        "Patch": 5
```

Comment:
Is this change minor or Patch? Also, the task version is already 0.0.5, please do the needful here

Reply:
I just bumped up the version which is being maintained. I don't know if you want to upgrade the minor one or not. Your call.